ext/DimensionalDataDiskArraysExt.jl

```diff
@@ -0,0 +1,18 @@
+"""
+    DimensionalDataDiskArraysExt
+
+Extend some methods of DiskArrays (`cache`, etc) to work on the base data of any DimArray.  
+"""
+module DimensionalDataDiskArraysExt
+
+using DimensionalData
+import DimensionalData: AbstractBasicDimArray
+import DiskArrays
+
+DiskArrays.cache(da::AbstractBasicDimArray; kwargs...) = rebuild(da; data = DiskArrays.cache(parent(da); kwargs...))
```

Comment:
```suggestion
DiskArrays.cache(x::Union{AbstractDimStack,AbstractArray}; kw...) = mondify(A -> DiskArrays.cache(A); kw...))
```

`modify` is more general here. Also `AbstractBasic DimArray` may not have a parent but AbstractDimArray will

Reply:
ah hold on, this is an invalidation....should we be doing this?

Reply:
or did you mean AbstractDimArray?

Reply:
I am almost certain it should be `AbstractDimArray`, didn't test but wouldn't this lead to StackOverflows otherwise?

Reply:
It was just a typo from editing on my phone, fixed now